packages/components/color-picker/src/index.vue

```diff
@@ -162,6 +159,7 @@ export default defineComponent({
   emits: ['change', 'active-change', UPDATE_MODEL_EVENT],
   setup(props, { emit }) {
     const { t } = useLocale()
+    const ns = useNamespace('color')
```

Comment:
Suggestion: there's no name called `color`.
```suggestion
    const ns = useNamespace('color-picker')
```